content/docs/index.mdx

```diff
@@ -15,34 +15,50 @@ In addition to the docs here, which are guides to building on Stellar, you'll al
 
 A rundown of what's here:
 
-## [Tutorials](./tutorials/create-account.mdx)
+## Tutorials
 
 If you’re new to Stellar and want to get an overview of the network, this is where to start. The early tutorials will show you how to do some basic things like create an account and make payments. The later tutorials cover more advanced topics like Stellar smart contracts. If you want to issue an asset or build an app, you're better served checking the sections dedicated to those paths.
 
-## [Issue Assets](./issuing-assets/index.mdx)
+<ReadMore url="./tutorials/create-account" />
```

Comment:
This seems fine, but it's a mild bummer that this isn't a "real" link that will work within github. Since we're not doing this frequently it seems okay though